boards/arm/olimexino_stm32f303rc/olimexino_stm32f303rc.dts

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) 2022 David Jablonski
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f3/stm32f303Xc.dtsi>
+#include <st/f3/stm32f303r(b-c)tx-pinctrl.dtsi>
+
+/ {
+	model = "Olimex OLIMEXINO-STM32F303RC board";
+	compatible = "olimex,olimexino_stm32f303rc", "st,stm32f303rc";
+
+	chosen {
+		zephyr,console = &usart2;
+		zephyr,shell-uart = &usart2;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,canbus = &can1;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		green_led_1: led_1 {
+			gpios = <&gpioa 5 GPIO_ACTIVE_HIGH>;
+			label = "Green LED";
+		};
+		yellow_led_2: led_2 {
+			gpios = <&gpioa 1 GPIO_ACTIVE_HIGH>;
+			label = "Yellow LED";
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		user_button: button {
+			label = "Boot/User";
+			gpios = <&gpioc 9 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	aliases {
+		led0 = &green_led_1;
+        led1 = &yellow_led_2;
```

Comment:
fix use of spaces here, replace with tabs.  Check file for any other cases.

Reply:
Thanks, done! Looks like the only case (:set tabstop=4, noexpandtab, %retab! in vim)